### ISSUE TITLE
[Tarea 1] Ability the rock

### ISSUE THREAD

**Issue by Pdesolmi** [IIC2113-2022-2/Syllabus#8]
"At the start of your turn, before your draw segment, you may take 1 card from your Ringside pile and place it on the bottom of your Arsenal."

Esto significa que elijo una carta de mi ringside, o saco la primera que está?

**Comment by Pdesolmi** [comment]
Y con respecto a esta: "At the start of your turn, before your draw segment, opponent must take the top card from his Arsenal and place it into his Ringside pile."

Esta habilidad se usa si o si? porque sale must, pero en el enunciado dice "Al comenzar el turno, el jugador PUEDE utilizar el superstar ability de su superestrella antes de robar una carta. Esto solo aplica para las superestrellas cuyos superstar
ability indican que solo se pueden utilizar “before your draw segment.”"

**Comment by Dnpoblete** [comment]
Hola @Pdesolmi, con respecto a tu primera pregunta, debes sacar la primera carta de la pila Ringside. Con respecto a la segunda pregunta es opcional si la usa o no.

Mucho éxito! ✨

**Comment by RodrigoToroIcarte** [comment]
No. La habilidad de THE ROCK consiste en **elegir** una carta de tu ringside y ponerla al final de tu arsenal. Por lo mismo, cuando robes NO saldrá la carta que pusiste en tu arsenal (porque esa carta quedó al final, no al principio).

Por ejemplo, digamos que tienes 5 cartas en tu ringside. De esas 5 el usuario elige una. Digamos que elige un "Punch". Entonces esa carta es puesta al final del arsenal. Por lo mimo, cuando THE ROCK robe una carta no debería ser el Punch que se recuperó. Debería ser otra carta (aunque existe la posibilidad que le salga un Punch si es que tiene varias copias de Punch en su mazo).

**Comment by RodrigoToroIcarte** [comment]
Respecto a la segunda pregunta, eso lo aclaré en el telegram. Efectivamente el efecto de Kane **NO** es opcional porque dice "must". Mientras que hay otras habilidades que sí son opcionales. Las habilidades de THE ROCK, JERICHO, STONE COLD y UNDERTAKER son opcionales. Las de KANE y MANKIND son obligatorias. HHH no tiene habilidad.

**Comment by Pdesolmi** [comment]
Gracias @RodrigoToroIcarte quedó claro.

**Comment by Pdesolmi** [comment]
última pregunta respecto a esas habilidades perdón, la de undertaker "Once during your turn, you may discard 2 cards to the Ringside pile and take 1 card from the Ringside pile and place it into your hand." Discard 2 cards se refiere a cards de la hand o no?

**Comment by RodrigoToroIcarte** [comment]
Pregunta con confianza no más :)

Sí. La habilidad del UNDERTAKER consiste en descartar dos cartas de tu mano (i.e., ponerlas en el ringside). Luego eliges una carta del ringside y la pones en tu mano.
